handlers/upload.go

```diff
@@ -101,6 +104,21 @@ func (r UploadVODRequest) IsProfileValid() bool {
 	return true
 }
 
+func (r UploadVODRequest) IsClipValid() bool {
+	startTime := r.ClipStrategy.StartTime
+	endTime := r.ClipStrategy.EndTime
+
+	if startTime < 0 || endTime <= 0 || startTime == endTime {
+		return false
+	}
+
+	epoch := time.Date(1970, 1, 1, 0, 0, 0, 0, time.UTC)
+	start := epoch.Add(time.Duration(startTime) * time.Second)
+	end := epoch.Add(time.Duration(endTime) * time.Second)
+
+	return !start.After(end)
```

Comment:
Could we just do `startTime < endTime` here?

Reply:
done